p2p/transport/websocket/websocket_test.go

```diff
@@ -15,6 +15,8 @@ import (
 	"testing"
 	"time"
 
+	mplex "github.com/libp2p/go-libp2p/p2p/muxer/mplex"
```

Comment:
do we need the rename?

Reply:
The package is named `peerstream_multiplex`, so I think we do.

Reply:
So, can we, like, fix that. Now?

Reply:
Probably.